go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/tektoncd/pipeline
 go 1.13
 
 require (
+	github.com/AlekSi/pointer v1.1.0 // indirect
```

Comment:
It looks like the test is using [`k8s.io/utils/pointer`](https://github.com/kubernetes/utils/blob/da695404a2bc/pointer/pointer.go#L114).  I'm not 100% sure but it doesn't look like this module is used.

Reply:
The code for the test file [is importing `k8s.io/utils/pointer`](https://github.com/tektoncd/pipeline/pull/3973/files#diff-2b91646885f5ea26b953eaab2bc73293cb29ba32a0de26c46d3f2521f558d8a5R30) for those funcs I think, isn't it?  I might be totally misunderstanding what's happening.

Reply:
Oh I didn't even see that ! Thanks for pointing it out, fixed 😄